app/views/layouts/mailer.html.erb

```diff
@@ -8,6 +8,17 @@
   </head>
 
   <body>
-    <%= yield %>
+
+    <%= render "layouts/nav" %>
+
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+
+    <div class="container">
+      <%= yield %>
+    </div>
+
+    <%= render "layouts/footer" %>
+
```

Comment:
The mailer layout is meant for email and as a result won't show the header/footer as they'll contain navigation and such.

For now I'd roll back your changes to this mailer layout and we'll worry about it a bit later when we have some emails to send out.